src/components/Tabs/presets/0-default.jsx

```diff
@@ -15,8 +15,7 @@ export default (
     tab9=""
     tab10=""
     activeTab={1}
-    size="xs"
-    border={false}
```

Comment:
Is removing the border variant intentional here?

Reply:
I have seen that the `border` is always set, so I thought it is unnecessary to have that property there.

![image](https://user-images.githubusercontent.com/86958774/170657559-5f2acbdd-6e2e-486a-b90a-7247406e0428.png)